packages/components/examples/project-file-t.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { signOut, T } from 'utils';
+import warningImg from 'assets/images';
+
+export function NotAuthorized() {
+  return (
+    <div className="container">
+      <T.H1 l10nId="NotAuthorized_oopsMessage">
```

Comment:
Is this demonstrating the localization of a different prefix? Maybe to encourage best practices we should avoid single letter names?

Reply:
https://github.com/udacity/fluent-react-utils/issues/17